translate.py

```diff
@@ -102,9 +102,32 @@ def report_score(name, score_total, words_total):
             name, score_total / words_total,
             name, math.exp(-score_total/words_total)))
 
+    def report_bleu():
+        import subprocess
+        print()
+        res = subprocess.check_output(
+            "perl tools/multi-bleu.perl %s < %s" % (opt.tgt, opt.output),
+            shell=True).decode("utf-8")
+        print(">> " + res.strip())
+
+    def report_rouge():
+        from tools.test_rouge import test_rouge
```

Comment:
the main code base is not allowed to import `tests` or `tools`. I think I would rather you just call through pipes as above.

Reply:
Where should I put the scripts in? ```Util.py``` ?

Reply:
No you can keep them in tools/ but call out to the command-line tools, just like you do for bleu. I don't want the eval in the main code-base.

Reply:
Get it. Thanks for your suggestions!